packages/python-packages/api-stub-generator/apistub/nodes/_function_node.py

```diff
@@ -146,13 +151,26 @@ def _parse_docstring(self):
                     kw_arg = kwargs[0]
                     self.args.remove(kw_arg)
                     self.args.append(kw_arg)
+            elif [x for x in self.args if x.argname == KW_ARG_NAME]:
+                # if kw arg details are not available in docstring and function has kwarg then flag it as error
+                self.errors.append("Keyword arg info is not available. Please ensure docstring has type info for all keyword args")
```

Comment:
This might not be an error - all API operations should have kwargs, but in some cases the kwargs will only be pipeline configuration - which wouldn't be documented on a given API.
It _would_ be an error if any API _didn't_ have `**kwargs` though.

The error for undocumented kwargs is much harder to parse - as we would actually have to parse the function code itself to see if kwargs is used anywhere, and extract those values to confirm they have a matching docstring. This would be a truly awesome feature in the future, though given its difficulty, I wouldn't think it's a short term item.